src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier color = new ColorSupplier();
+        ball.setColor(color.getRandomColor());
+        Random random = new Random();
```

Comment:
same, lets make Random and ColorSupplier as class level variables, not to create new instances every time we call getRandomBall() method